.travis.yml

```diff
@@ -21,9 +21,6 @@ jobs:
     - jdk: openjdk11
       env: MULTITHREADED=true
       name: "Java 11 w/ multithreaded"
-    - stage: puppetserver vulnerability scan
-      script:
-        - lein nvd check
```

Comment:
Should we also remove the plugin from project.clj and remove the suppression file?